app/serializers/weather_serializer.rb

```diff
@@ -0,0 +1,8 @@
+class WeatherSerializer
+  include FastJsonapi::ObjectSerializer
+  attributes :id, :description, :humidity, :temperature, :daily
+
+  # attribute :daily do |object|
```

Comment:
Might want to remove this commented out code

Reply:
Changing it now. I'll push in a bit